cocoa/src/toga_cocoa/window.py

```diff
@@ -175,6 +175,11 @@ def __init__(self, interface, title, position, size):
         self.container = Container(on_refresh=self.content_refreshed)
         self.native.contentView = self.container.native
 
+        # In app full-screen mode, window._impl.native.contentView is moved to a
+        # _NSFullScreenWindow, making NSWindow.contentView as None. Hence, retain a
+        # reference to the view before going full-screen.
+        self.content_view = self.native.contentView
```

Comment:
Full explaination comment that I had previously written, but felt too big to include in the source file:

Once we go fullscreen on `window._impl.native.contentView`, the view referenced by `window._impl.native.contentView` is detached from the current native `NSWindow`, and attached to a `_NSFullScreenWindow`. And the view can remain attached to 1 native window at a time i.e., either to `NSWindow` or `NSFullScreenWindow`, but not to both. 

Hence, we can no longer accesss the view by `window._impl.native.contentView`, since `window._impl.native.contentView` will be None. So, it is useful to retain a reference to the view attached to `window._impl.native.contentView`, before going fullscreen or doing any other operation that would displace the `contentView` from the current window.

Reply:
You may not be able to reference `_impl.native.contentView`, but you *can* reference `_impl.container.native`. That keeps the container as a source of truth, and removes any bugs that could leak in if the container content is changed but `content_view` isn't updated.